src/vm/excep.h

```diff
@@ -24,6 +24,7 @@ class Thread;
 #include "interoputil.h"
 
 BOOL IsExceptionFromManagedCode(const EXCEPTION_RECORD * pExceptionRecord);
+bool IsIPExcluded(UINT_PTR uControlPc);
```

Comment:
Maybe nice to give it more descriptive name now that it is used from multiple places.